docs/en/latest/debug-mode.md

```diff
@@ -0,0 +1,137 @@
+---
+id: debug-mode
+title: Debug mode
+keywords:
+  - API gateway
+  - Apache APISIX
+  - Debug mode
+description: Guide for enabling debug mode in Apache APISIX.
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+You can use APISIX's debug mode to troubleshoot your configuration.
```

Comment:
```suggestion
You can use APISIX's debug mode to troubleshoot your configurations.
```

Reply:
A deployment of APISIX will have a configuration. Considering a configuration for APISIX as a whole. Wouldn't it be better to keep it singular itself?

Reply:
Yeah you are right.